cms/templates/settings.html

```diff
@@ -294,8 +294,14 @@ <h2 class="title-2">${_("Course Enrollment Email")}</h2>
             <span class="tip">${_("Settings for emails that students receive upon course enrollment")}</span>
           </header>
           <section class="group-settings">
-            <p><strong>By default, this feature is disabled.</strong> You can either use the default email content or create your own.<br />
-              Students enrolling right on the start date will receive the post-course-start email.<br />
+            <p>${_("{strong_start}By default, this feature is disabled.{strong_end} You can either use the default email content or create your own. "
```

Comment:
If we're already breaking these long lines, we could wrap these a bit sooner and make the lines a bit shorter, especially the second one here.